.husky/pre-commit

```diff
@@ -1,4 +1,5 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
+npm run lint
```

Comment:
Below we split out lint and test into two different steps. This aligns to our standard setup.